changelog.d/12600.misc

```diff
@@ -0,0 +1 @@
+Explicitly select the `flake8` lints used.
```

Comment:
I think this needs clarification too, but currently I'm not quite clear what the actual change is so can't really suggest an improvement.